core/src/main/java/io/aiven/klaw/service/MailUtils.java

```diff
@@ -715,4 +542,263 @@ public String sendMailToSaasAdmin(int tenantId, String userName, String period,
+    TOPIC_REQUEST_DENIED(
+        "Topic Request Denied",
+        (manageDatabase, tenantId, mailInfo) ->
+            String.format(
+                manageDatabase.getKwPropertyValue(TOPIC_REQ_DENY_KEY, tenantId),
+                "'" + mailInfo.topicName + "'",
+                "'" + mailInfo.reasonToDecline + "'"),
+        false),
+    CONNECTOR_REQUEST_DENIED(
+        "Connector Request Denied",
+        m ->
+            "Connector Request on "
+                + m.topicName
+                + " denied by "
+                + m.approverUsername
+                + "because : "
+                + m.reasonToDecline),
+    ACL_REQUESTED(
+        "New Acl Request",
+        (manageDatabase, tenantId, mailInfo) ->
+            String.format(
+                manageDatabase.getKwPropertyValue(ACL_REQ_KEY, tenantId),
+                "'" + mailInfo.acl + "'",
+                "'" + mailInfo.topicName + "'"),
+        true),
+    ACL_DELETE_REQUESTED(
+        "Acl Delete Request",
+        (manageDatabase, tenantId, mailInfo) ->
+            String.format(
+                manageDatabase.getKwPropertyValue(ACL_DELETE_REQ_KEY, tenantId),
+                "'" + mailInfo.acl + "'",
+                "'" + mailInfo.topicName + "'"),
+        true),
+    ACL_REQUEST_APPROVED(
+        "Acl Request Approved",
+        (manageDatabase, tenantId, mailInfo) ->
+            String.format(
+                manageDatabase.getKwPropertyValue(ACL_REQ_APPRVL_KEY, tenantId),
+                "'" + mailInfo.acl + "'",
+                "'" + mailInfo.topicName + "'"),
+        false),
+    ACL_REQUEST_APPROVAL_ADDED(
+        "Acl Request Approval Addded",
+        m ->
+            String.format(
+                "Acl Request %s for topic %s has had an approval added to it.",
+                "'" + m.acl + "'", "'" + m.topicName + "'")),
+
+    ACL_REQUEST_DENIED(
+        "Acl Request Denied",
+        ((manageDatabase, tenantId, mailInfo) ->
+            String.format(
+                manageDatabase.getKwPropertyValue(ACL_REQ_DENY_KEY, tenantId),
+                "'" + mailInfo.acl + "'",
+                "'" + mailInfo.topicName + "'",
+                "'" + mailInfo.reasonToDecline + "'")),
+        false),
+    ACL_REQUEST_FAILURE(
+        "Request processing failed.",
+        m -> "Acl Request processing failed : " + m.acl + ", " + m.topicName),
+    SCHEMA_REQUESTED("New Schema Request", m -> "New Schema Request on " + m.topicName, true),
+    SCHEMA_REQUEST_APPROVED(
+        "Schema Request Approved",
+        m -> "Schema Request on " + m.topicName + " approved by " + m.approverUsername),
+    SCHEMA_REQUEST_DENIED(
+        "Schema Request Denied",
+        m ->
+            "Schema Request on "
+                + m.topicName
+                + " denied by "
+                + m.approverUsername
+                + "because : "
+                + m.reasonToDecline),
+    REGISTER_USER_REQUEST("TODO", (Function<MailInfo, String>) null),
```

Comment:
No usages of this were found, should we just remove it?

Reply:
Yeah you can remove it, its called from another area (that i literally was looking at a few hours ago because of a seperate bug Murali fixed )

Reply:
ok, thanks for confirmation, removed